runtime/src/main/java/org/corfudb/runtime/view/replication/QuorumReplicationProtocol.java

```diff
@@ -7,10 +7,8 @@
 package org.corfudb.runtime.view.replication;
 
 import java.time.Duration;
-import java.util.AbstractMap;
-import java.util.Comparator;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
expand. 

Also: https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports

Reply:
Done. Thanks for this tip